server/src/test/meal.js

```diff
@@ -0,0 +1,93 @@
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import app from "../app";
+
+chai.use(chaiHttp);
+chai.should();
+
+
+describe("meal", () => {
+  describe("GET Meal", () => {
+    it("should get all meals for the day", done => {
+      chai
+        .request(app)
+        .get("/api/v1/meals")
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a("object");
+          done();
+        });
+    });
+
+    it("should GET a single Meal", done => {
+      const id = 1;
+      chai
+        .request(app)
+        .get(`/api/v1/meals/${id}`)
+        .end((err, res) => {
+          res.should.have.status(404);
+          res.body.should.be.a("object");
```

Comment:
You want to write more robust assertions. Assert that the properties in the response.body object are exactly what they should be